herbert_core/classes/instrument_classes/instrument/detectors/@IX_detector_array/IX_detector_array.m

```diff
@@ -146,8 +199,70 @@
         end
         
         %------------------------------------------------------------------
+        
+        function detpar = convert_to_old_detpar(obj)
+            detpar = struct();
+            if size(obj.det_bank.id,1)==1
```

Comment:
Can check `isscalar` or `isvector`? Or does it need to be checked like this explicitly?

Reply:
will revisit after merge